docs/dashboards/data-modifiers.md

```diff
@@ -0,0 +1,80 @@
+Data Modifiers
+===
+
```

Comment:
I would add why it's beneficial to use the modifiers instead of parsing everything on your own.